src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -253,19 +256,20 @@ def print_all_container_logs_in_namespace(ns)
 
 # Wait for a cf service asynchronous operation to complete.
 def wait_for_async_service_operation(service_instance_name, retries=0)
+    duration = 5
+    STDERR.puts "#{c_bold}# Waiting for service instance #{service_instance} to complete operation, at most #{retries} retries, in increments of #{duration}s#{c_reset}"
```

Comment:
That seems rather verbose…
Do we really need the bold here? Or dumping it on `STDERR` (potentially moving it around relative to all the other text)?

Reply:
Will compress. (Tomorrow).
Wrt bold, see previous.
Wrt `STDERR`, it was chosen so that the output is properly positioned relative to the output of the `run` commands, which also posts to `STDERR`.

I am willing to say that everything should go to `STDOUT`, and make that change.

Reply:
Done with commit e8a0eca19f6e6b69f9f685bda659c9a87380d02c.